docs/src/tutorials/RK_order_conditions.md

```diff
@@ -0,0 +1,82 @@
+# Runge-Kutta Order conditions
+
+In this tutorial, we generate order conditions for a generic explicit Runge-Kutta method.
+
+First, we create symbolic coefficient arrays with the appropriate structure.
+
+
+```julia
+using BSeries, SymPy, Latexify
+
+s = 4  # Stages
+p = 4  # Desired order of accuracy
+
+A = Array{Sym,2}(undef,s,s)
+b = Array{Sym,1}(undef,s)
+c = Array{Sym,1}(undef,s);
+```
+
+
+```julia
+for i = 1:s
+    b[i] = symbols("b$i", real=true)
+    for j = 1:i-1
+        A[i,j] = symbols("a$i$j", real=true)
+    end
+    for j = i:s
+        A[i,j] = 0
+    end
+end
+
+for i = 1:s
+    c[i] = 0
+    for j = 1:i-1
+        c[i] += A[i,j]
+    end
+end
+```
```

Comment:
Continuing the examples block structure from above

````suggestion
```@example bseries-RK-order-conditions
for i in 1:s
    b[i] = symbols("b$i", real = true)
    for j in 1:i-1
        A[i, j] = symbols("a$i$j", real = true)
    end
    for j in i:s
        A[i, j] = 0
    end
end

for i in 1:s
    c[i] = 0
    for j in 1:i-1
        c[i] += A[i, j]
    end
end
```
````